atecc608a/atecc508a_config_dev.h

```diff
@@ -148,7 +148,11 @@ const uint8_t template_config_508a_dev[]  =
   0x13, 0x00, /* 104-105 KeyConfig slot 4 */
   0x13, 0x00, /* 106-107 KeyConfig slot 5 */
   0x13, 0x00, /* 108-109 KeyConfig slot 6 */
-  0x13, 0x00, /* 110-111 KeyConfig slot 7 */
+  /* Slot 7 is used to contain ECDH master secret, so according to the KeyType description,
```

Comment:
I suggest we write `ECDH shared secret` here.

(Master secret is TLS terminology and can be acquired many different ways depending on the ciphersuite. ECDH is a key exchange algorithm where the parties end up in the possession of a secret known only to them. It might be called a key exchange algorithm, but using the resulting secret directly as a key is not the best practice.)